server/norace_test.go

```diff
@@ -4196,3 +4196,82 @@ func TestNoRaceJetStreamStreamInfoSubjectDetailsLimits(t *testing.T) {
 		t.Fatalf("Did not get correct error response: %+v", sir.Error)
 	}
 }
+
+func TestJetStreamSparseConsumers(t *testing.T) {
```

Comment:
Prefix should be `TestNoRace...` if you want it to be run on Travis (since we run the "no race" tests with `-run=TestNoRace ..`

Reply:
Your right, will fix.